### ISSUE TITLE
Unknown Packet 6F45 and F180

### ISSUE THREAD

**Issue by WangOldFive1** [OpenKore/openkore#532]
First error kept changing, but the 2nd one is always F180

**Comment by Novishivsmoke** [comment]
Extract the keys using PEEK, then back to talk with us !

**Comment by WangOldFive1** [comment]
Sorry, don't know what PEEK is and not sure how to do it.

**Comment by Novishivsmoke** [comment]
Please. 

http://forums.openkore.com/viewtopic.php?f=36&t=207278

Provide more information about this error.

**Comment by WangOldFive1** [comment]
Hi,

I download PEEK, but it doesn't show Start Analyze.
It shows Start Recorder and Extract Length.
I located my Ragexe.exe and click on Extract Length.
It shows an error message "DllGen: Packet Key Pattern not found"

I also tried using Packet Extract on Ragexe.exe
The message I got is Search of Packet Len Map failed!

I downloaded Openkore 2.05.1
The server.txt was wrong.
So I located Openkore master.
The server.txt is correct, but I have a new error message when running
under XKore 1.
NetRedirect.dll can not inject

If I run under XKore 2, after I opened RO, XKore 2 ask for my id and
password and it becomes double login and disconnects.
Then when it relog, it show unknown packet issue 082D.

If I run either one without XKore, then I run into packet problem.

I tried to fix the 082D issue, then other packet problem came out.

I also tried to use Poseidon, but not sure how to do it correctly.
